packages/phoenix-app/package.json

```diff
@@ -12,15 +12,16 @@
   "scripts": {
     "ng": "ng",
     "start": "ng serve",
-    "start:ssl": "ng serve --ssl",
+    "start:ssl": "ng serve --ssl --host=0.0.0.0",
     "build": "ng build",
     "test": "ng test",
     "lint": "ng lint",
     "e2e": "ng e2e",
     "deploy": "mv ../../docs/api-docs ../../api-docs && ng build --prod --output-path ../../docs --base-href \"/phoenix/\" && cp ../../docs/index.html ../../docs/404.html && mv ../../api-docs ../../docs/api-docs",
     "test:coverage": "rm -rf ./coverage && ng test --no-watch --code-coverage",
     "coveralls": "cat ./coverage/phoenix/lcov.info | ./node_modules/coveralls/bin/coveralls.js",
-    "version": "npm run deploy"
+    "version": "npm run deploy",
+    "deploy:atlas": "ng build --prod --output-path ./atlas --base-href \"/hsf-phoenix/atlas/\" && cp ./docs/index.html ./docs/404.html"  
```

Comment:
I think it's better if we generalize this script for deploying anywhere.
We can make this:

```
"deploy:web": "ng build --prod --output-path ./docs",
"postdeploy:atlas": "cp ./docs/index.html ./docs/404.html"
```

And then we can define the `--base-href` when calling the script:

```
npm run deploy:web -- --base-href "//hsf-phoenix/atlas/"
```

(You might need a single `/` instead of `//`. Using `//` at start here because git bash somehow converts to a full windows path otherwise)

Reply:
This single script works for any path (I checked on multiple paths on Apache). We can just run this command and put it anywhere on the server and it will work.

```
"deploy:web": "ng build --prod --output-path ./docs --base-href \"./\" && cp ./docs/index.html ./docs/404.html"
```